static/js/portico/integrations.js

```diff
@@ -371,11 +371,32 @@ function integration_events() {
     });
 }
 
+function sync_open_organizations_page_with_current_hash() {
+    const hash = window.location.hash;
+    if (!hash || hash === "#all" || hash === "#undefined") {
+        $(".eligible_realm").show();
+        $(".realm-category").removeClass("selected");
+        $(`[data-category="all"]`).addClass("selected");
+    } else {
+        $(".eligible_realm").hide();
+        $(`.eligible_realm[data-org-type="${CSS.escape(hash.slice(1))}"]`).show();
+        $(".realm-category").removeClass("selected");
+        $(`[data-category="${CSS.escape(hash.slice(1))}"]`).addClass("selected");
+    }
+}
+
 // init
 $(() => {
-    integration_events();
-    load_data();
     dispatch("LOAD_PATH");
-    $(".integrations .searchbar input[type='text']").trigger("focus");
-    adjust_font_sizing();
+    if (window.location.pathname.includes("communities")) {
+        sync_open_organizations_page_with_current_hash();
+        $(window).on("hashchange", () => {
+            sync_open_organizations_page_with_current_hash();
+        });
```

Comment:
I think it'd likely be better to just put the JS for the new page in a new file `communities.js`, and use `entrypoint="communities"` to reference it -- it's not obvious we want to share any JS with the other page?

(Or is `dispatch()` here doing something useful?)

Reply:
@timabbott do I need to do anything else for the new `communities.js` file to be picked up by webpack? It is not working after doing the above changes. 

```
  File "/home/aman/zulip/templates/zerver/base.html", line 29, in block 'webpack'
    {% for filename in webpack_entry(entrypoint) -%}
  File "/home/aman/zulip/zerver/lib/templates.py", line 199, in webpack_entry
    for filename in stats["chunks"][entrypoint]
KeyError: 'communities'
```

I tried clearing webpack cache but no success.

Reply:
Done!